Automattic/newspack-plugin
# Stripe billing portal link

### All Submissions:






### Changes proposed in this Pull Request:



Adds an integration with WooCommerce's "My Account" page when Stripe is set as Reader Revenue platform.

The new "Billing" page only contains a link to Stripe's billing portal, and it's pretty bland:

<img width="691" alt="image" src="https://user-images.githubusercontent.com/7383192/177945403-49ae26e5-ec13-47e2-b689-fe7b0b60f2d8.png">

Any thoughts here, @thomasguillot ?

In the future, we might implement such a billing portal on-site (https://github.com/Automattic/newspack-plugin/issues/1764).

Closes #1742
Closes #1739
Closes #1740
Closes #1741
Closes #1782 

### How to test the changes in this Pull Request:

1. Switch Newspack Theme to https://github.com/Automattic/newspack-theme/pull/1875
1. Ensure that required WooCommerce plugins are active, Stripe is configured as the Reader Revenue platform, and site is reachable for webhooks 
3. Ensure that `NEWSPACK_EXPERIMENTAL_READER_ACTIVATION` flag is set to `true`
4. As a non-logged-in and non-existing user, make a donation
5. Observe that `/my-account` page redirects to "Account Details"
6. Observe three links are available in "My Account" nav – "Account Details", "Billing", and "Logout"
7.  Go to "Billing" and click the "Update payment details" link – observe it opens Stripe customer portal, where you can update the CC details, cancel and edit subscriptions, and see billing history
9. Observe that password reset link can be sent in "Account Details" part in order to set up or reset password

### Other information:
